nginx_proxy/nginx.conf

```diff
@@ -10,6 +10,7 @@ http {
     map $http_upgrade $connection_upgrade {
         default upgrade;
         ''      close;
+    server_names_hash_bucket_size 64;
```

Comment:
Indenting is off.

Secondly, why not add this to the `http` level of the configuration?

Reply:
You're correct, this *should* be in the `http` level of the configuration, not where I've put it. I accidentally put it within the map brackets, hence why the indentation is also off. My fault, tried using the Github online editor since it was such an easy fix and messed up.